### DISCUSSION TITLE
Unable to setup the resend  custom domain with vercel  -  Vercel and resend

### DISCUSSION THREAD

**Discussion by ceo-szhoes** [vercel/community#7371] [Help] [answered]
### Summary

I am a beginner working with the resend  for email hosting. 
I have tried to create a custom domain in resend and add the TXT and MX record to the vercel dns for my hosted project. Then start the verification process in resend. But every time it get failure, I have also try multiple time to create new domain and configure with vercel. But every time to get **failure**, 
![Screenshot from 2024-07-05 17-17-41](https://github.com/vercel/community/assets/171259998/12d2c6f2-dd4a-4147-b0f8-b87999b879c6)
![Screenshot from 2024-07-05 17-17-24](https://github.com/vercel/community/assets/171259998/6e77de38-f5b6-4654-beda-632fee078fea)
![Screenshot from 2024-07-05 17-22-39](https://github.com/vercel/community/assets/171259998/28fd4d61-60da-441d-a15e-3ca8e16920fb)

### Steps to Reproduce

1. Create a custom domain in resend
2. Add TXT and MX record in vercel project
3. Start the verification process in resend

**Comment by pawlean** [comment]
Hi @ceo-szhoes,

How long ago was this? You may just have to wait a while for it to propagate.

**Reply by ceo-szhoes** [reply]
Hi @pawlean,
Thanks for the reply,

I have tried it from 1 week, and also waited for 3 days

**Reply by pawlean** [reply]
Some things to try:

**Verify DNS Record configuration:** Are the TXT and MX records are correctly added to your Vercel DNS settings. Double-check the values provided by Resend.

**Debugging Steps:** Use DNS lookup tools (like [DNSChecker](https://dnschecker.org/)) to verify that the TXT and MX records are correctly propagated. Check for typos or incorrect entries in your DNS records.

**Propagation time:** DNS changes can take up to 48 hours to propagate.

**Reply by ceo-szhoes** [reply]
Hi, @pawlean 
I have tried it all but doesn't work for me.

I have check the record and also tried to debug it but they don't work for me. Do any other option is there to configure resend.
![Screenshot from 2024-07-06 15-06-16](https://github.com/vercel/community/assets/171259998/a639fb49-767d-4e71-976d-03e50649f2b3)

**Reply by pawlean** [reply]
@ceo-szhoes I'm confused by your screenshot, which domain did you purchase and trying to use on Vercel?

**Reply by ceo-szhoes** [reply, marked as answer]
Thanks for you help. I got the solution. The error is of CNAME configuration

[a comment by an automated account is not shown]
